lms/djangoapps/course_blocks/transformers/user_partitions.py

```diff
@@ -42,7 +42,11 @@ def collect(cls, block_structure):
         # Because user partitions are course-wide, only store data for
         # them on the root block.
         root_block = block_structure.get_xblock(block_structure.root_block_usage_key)
-        user_partitions = getattr(root_block, 'user_partitions', []) or []
```

Comment:
@nasthagiri , was there any reason for "`or []`" in this line?
as `{} or [] = []`.